inventables/rentables
# Update rails minor version to prevent mimemagic error

### What does this PR do?

This PR updates the Rails version of this repo to [Rails 5.2.5](https://weblog.rubyonrails.org/releases/) to prevent build failure due to a dependency on Mimemagic.

### Background Context

We've been dealing with this in other places as well. Long story short, a gem that Rails depends on was pulled due to licensing issues. This would prevent anyone from downloading and running rentables. Rails put out a fix in a minor version update. 

### What steps did you take to test your changes?

Because rails was already pinned to 5.2, this update is minor. After making this change, I followed the setup steps from the README:

- Clone the git repo and navigate to code directory
- Install the gems: `bundle install`
- Run migrations: `rake db:migrate`
- Seed the database: `rails db:seed`
- Start the server: `rails server`

Then, I started my server and confirmed that it ran with the seeded data:

<img width="813" alt="image" src="https://user-images.githubusercontent.com/46462767/115061543-ef3a3700-9eae-11eb-8585-7f13155d89f6.png">

I also ran the test suite and it's behaving as expected:

<img width="828" alt="image" src="https://user-images.githubusercontent.com/46462767/115061738-29a3d400-9eaf-11eb-8e69-3262d86989eb.png">
